mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIndexer.java

```diff
@@ -53,6 +55,29 @@ public String getId() {
         public String getUpdatedDocumentPart() {
             return updatedDocumentPart;
         }
+
+        @Override
+        public final boolean equals(Object o) {
```

Comment:
It's a known one but the added complexity is not worth it (at least it's the implicit team choice we've made).